Add new toasts examples

mdo opened the pull request:
This should cover us nicely for how to build and customize toasts. This also basically includes how to do a dark mode (use `.bg-dark`) version.

- Adds a new custom content example that removes the toast header and adds a custom close icon
- Adds an example for adding content, like more buttons
- Adds docs for how to create your own color schemes

Fixes #28201 and fixes #28202.

Preview:

* <https://deploy-preview-30572--twbs-bootstrap.netlify.app/docs/5.0/components/toasts/#custom-content>
* <https://deploy-preview-30572--twbs-bootstrap.netlify.app/docs/5.0/components/toasts/#color-schemes>

mdo committed:
Add two new toasts examples

- Adds a new custom content example that removes the toast header and adds a custom close icon
- Adds docs for how to create your own color schemes
- Adds example of additional buttons

XhmikosR commented on site/content/docs/4.3/components/toasts.md:
```diff
@@ -98,6 +98,60 @@ When you have multiple toasts, we default to vertically stacking them in a reada
+  </div>
+  <button type="button" class="ml-auto p-2 close" data-dismiss="toast" aria-label="Close">
+    <span aria-hidden="true">
+      <svg class="bi bi-x" width="1em" height="1em" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```
We should use what we currently use everywhere, `&times;`

mdo replied:
The point of the example here is to show a different icon :).

XhmikosR replied:
I'm 50-50 on this. I mean, it's generally pretty straight-forward, I'm not sure it needs an example...

XhmikosR replied:
Also, I'd say the extra `span` is not needed; you should move `aria-hidden` to the SVG itself.

mdo committed:
Update toasts examples

- Remove span and place aria on SVG element
- Add .border-0 to color scheme example for crisper edges

mdo committed:
Merge branch 'main' into toast-options

XhmikosR committed:
Update toasts.md

XhmikosR committed:
Merge branch 'main' into toast-options

XhmikosR commented:
Just squash and merge @mdo, but it LGTM
